src/chainstate/burn/db/sortdb.rs

```diff
@@ -1038,6 +1109,37 @@ impl<'a> SortitionHandleTx<'a> {
         }
     }
 
+    fn get_blocks_with_stacks_txs_at_tip(&mut self, next: bool) -> Result<u64, db_error> {
+        let chain_tip = self.context.chain_tip.clone();
+        self.get_blocks_with_stacks_txs(next, &chain_tip)
+    }
+
+    ///
+    /// Returns the number of blocks in this Sortition fork with Stacking-related
+    ///   bitcoin operations.
+    ///  the `next` parameter controls whether or not to return the value used for the
+    ///   block chosen in the current sortition or the value that should be used for blocks
+    ///   in the subsequent sortition
+    ///
+    pub fn get_blocks_with_stacks_txs(
```

Comment:
Not a bug, but more of a lint/readability concern.  Can this method be called `get_num_blocks_with_stacks_txs` instead, so as to clarify that it is returning a count instead of the actual blocks?  Also, might I suggest creating two methods to remove the need for `next` (whose usage isn't obvious) -- `get_num_blocks_with_stacks_txs` and `get_next_num_blocks_with_stacks_txs`?

Reply:
I'm also unclear why `next` is needed?  Maybe I need to re-read this code a few times.

Reply:
Next is necessary because StackStxOps are validated in the burnchain block that they are included in, but not _processed_ until subsequent blocks.